eng/common/scripts/modules/ChangeLog-Operations.psm1

```diff
@@ -1,5 +1,5 @@
 # Common Changelog Operations
-
+# This Files has been retired
```

Comment:
Is this for testing or is this a change you are planning to add?

Reply:
Yes this is for testing. I will clean it up before merging.